v2/google-ads-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/templates/GoogleAdsToBigQuery.java

```diff
@@ -99,8 +99,7 @@ public interface GoogleAdsToBigQueryOptions extends WriteOptions, GoogleAdsOptio
     @TemplateParameter.Text(
         order = 3,
         description = "Google Ads Query Language query",
-        helpText =
-            "The query to use to get the data. See Google Ads Query Language. For example: `SELECT campaign.id, campaign.name FROM campaign`.",
+        helpText = "The query to use to get the data. See Google Ads Query Language.",
```

Comment:
Should this have a link to the Google Ads Query Language page?

https://developers.google.com/google-ads/api/docs/query/overview